pkg/segment/structs/segmeta.go

```diff
@@ -24,8 +24,9 @@ import (
 const MAX_SEGMETA_FSIZE = 10_000_000 // 10 MB
 
 type ColSizeInfo struct {
-	CmiSize uint64 `json:"cmiSize"`
-	CsgSize uint64 `json:"csgSize"`
+	CmiSize  uint64 `json:"cmiSize"`
+	CsgSize  uint64 `json:"csgSize"`
+	CValSize int8   `json:"cValSize"`
```

Comment:
maybe we can call this `ConsistentCvalSize` .. something that denotes not the size of the whole column values but the consistency of the columnVal size across the whole segment